recipes/magicgui/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "magicgui" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 65e7b6845e636a302aecf2e736232aba7f9651ea782230ab4bb58627d957e893
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+    - qtpy >=1.7.0
```

Comment:
Do you really need qtpy at build time?

Reply:
Hi @scopatz,

thank you very much for spotting this. I don't know how it ended up there :)